### PR TITLE
Adds additional release info, including a11y, events, system upgrades

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3172](https://openedx.atlassian.net/browse/DOC-3172)

Further digging turned up these additional features that might be worth including in the Eucalyptus release notes. Feel free to mark any of them as unnecessary. 
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @jbarciauskas 
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check/copy edit/dev edit): @catong @lamagnifica @pdesjardins @srpearce
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @shaunagm 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 1]
```diff
@@ -86,18 +88,207 @@ Peer Instruction XBlock
...
+that is presented by the the edX e-commerce service. The functionality of this
+page has not changed, and learners can continue to check out using their credit
+cards or PayPal. For more information, see :ref:`installation:Adding ECommerce
+to Open edX`.
+
+===============================
+Badges for Custom Course Events
+===============================
+
+Open edX instances can now award badges to learners for certain milestone
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Suggestion: "award badges" -> "create badges" or "create and award badges". (Don't want to give the impression that these badges already exist.)

**Comment by jbarciauskas** [comment]
👍

**Comment by lamagnifica** [comment]
@catong ready for your re-review

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 2]
```diff
@@ -20,50 +20,34 @@ The following new features are included in the Open edX Eucalyptus release.
  :depth: 1
  :local:
 
-===================================================
+================================
 Self-Paced Course Configuration
-===================================================
+================================
 
 When course teams create a course, they can can now select the way a course is
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Duplicated word "can"

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 3]
```diff
@@ -20,50 +20,34 @@ The following new features are included in the Open edX Eucalyptus release.
...
-=====
-Teams
-=====
-
-With the teams feature, a learner can browse topics and then create a team to
-take part in group activities within the chosen topic. Teams provide small
-group interactions and projects in your course. For more information, see
-:ref:`opencoursestaff:CA_Teams_Overview`.
+delivered to learners. A schedule can be set for the course, including due
+dates for assignments or exams, or the course can be self-paced, which allows
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: I'd reword this so that we don't have to switch to a passive construction. How about: "You can set a schedule for the course, including due dates for assignments or exams, or you can specify that the course is self-paced, which allows..."

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 4]
```diff
@@ -86,18 +88,207 @@ Peer Instruction XBlock
...
+===================================================
+
+This release adds the following options for course teams to grade learner
+responses in open response assessments.
+
+* In Studio, course teams can include a staff assessment step as part of the
+  assignment.
+
+* In the LMS, course teams can assess any individual learner’s response at any
+  time to override their assignment grade.
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Nitty, but possibly make it more clear that it's not just simply entering some override grade, but that staff actually perform grading (using the same rubric), and that this staff grade overrides all other grades.

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 5]
```diff
@@ -126,13 +352,104 @@ In keeping with edX's commitment to creating accessible content for everyone,
...
+* The OAuth 2.0 provider library has been updated to version 0.5.0. This update
+  includes support for the client credentials grant.
+
+* The number of days before OAuth tokens expire for mobile OAuth clients can
+  now be configured. For more information, see :ref:`installation:Configure
+  OAuth Token Expiration Days`.
+
+* In this release, Ruby has been removed from the edx-platform repository. As a
+  result, the ``prereqs`` paver command no longer installs the prerequisite
+  environment for Ruby, and the edxapp user on new instances of Devstack will
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: "will no longer have" -> "no longer has"

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 6]
```diff
@@ -126,13 +352,104 @@ In keeping with edX's commitment to creating accessible content for everyone,
...
+  no longer have ``rbenv`` in the search path. However, discussion forums use
+  Ruby, and the forums user continues to have ``rbenv`` in the search path.
+
+* A new option for enabling or disabling the bulk email feature is available on
+  the Django administration console. You can now use this option to change a
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: I don't understand this sentence: "You can now use this option to change a setting rather than editing different configuration files." The first sentence makes the option sound like a simple enable/disable switch. Perhaps change "You can now use this option to change a setting" to something specifically about enabling/disabling bulk email if that's all, or rewrite the first sentence to be about settings for bulk email being consolidated in Django admin rather than being in various separate files?

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 7]
```diff
@@ -126,13 +352,104 @@ In keeping with edX's commitment to creating accessible content for everyone,
...
+  setting rather than editing different configuration files. For more
+  information, see :ref:`openreleasenotes:Enable Bulk Email`.
+
+  ..important:: As a result of this change, the bulk email feature is disabled
+    until you manually re-enable it.
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Just for new courses or existing courses too?

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 8]
```diff
@@ -154,23 +471,31 @@ deprecated.
 For more information about supported and deprecated web services,
 see :ref:`openplatformapi:Open edX Platform APIs`.
 
+==================================
+Deprecated Tools and Problem Types
+==================================
+
+* The randomize component is now deprecated. To provide randomized content, use
+  the Content Libraries feature to create randomized content blocks. For more
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: lowercase "content libraries feature"
Or, better than referring to a feature by name, just talk about the functionality. Something like:  "To provide randomized content in your courses, add randomized content blocks that randomly assign problems from content libraries."

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 9]
```diff
@@ -154,23 +471,31 @@ deprecated.
...
+  information, see :ref:`opencoursestaff:Randomized Content Blocks`.
+
+* The original drag and drop problem type is now deprecated. A new mobile-
+  ready, accessible drag and drop problem type is now available. For more
```
Comment (catong) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Perhaps strike the second occurrence of "now" before "available".

**Comment by catong** [comment]
@lamagnifica Sorry, I ended up doing a wider editing pass than just re-reviewing my previous edit. LGTM 👍
